### DISCUSSION TITLE
Come talk about publiccode.yml at OSPOlogy Live?

### DISCUSSION THREAD

**Discussion by ElenaFdR** [publiccodeyml/publiccode.yml#182] [General]
Hello publiccode.yml community,
Is there anyone who'd be willing to come give a short talk about publiccode.yml and what's next for your community/project at [OSPOlogy Live](https://community.linuxfoundation.org/events/details/lfhq-ospology-european-chapter-presents-ospologylive-apeldoorn/) in Apeldoorn, the Netherlands, on 28-29 February?
This OSPOlogy workshop is focused on open source in the public sector, and we (the conference organizers) think the audience would enjoy learning about your project.
Please email me at elena @ publiccode.net.
Thanks!

Elena

**Comment by bfabio** [comment]
@ElenaFdR thanks for the invitation, I just sent you an email
